### PR TITLE
Follow up on #156

### PR DESCRIPTION
**Pull request by Girgias**
A follow up to #156 and some rewording/restructuring for some pages

### CODE REVIEW

File: language/constants.xml [hunk 1]
```diff
@@ -57,8 +58,8 @@ define("__FOO__", "something");
   </note>
 
   <simpara>
-   Like &link.superglobals;, the scope of a constant is global.  You 
-   can access constants anywhere in your script without regard to scope.  
+   Like &link.superglobals;, the scope of a constant is global.
+   Constants can be accessed from anywhere in a script without regard to scope.
    For more information on scope, read the manual section on
    <link linkend="language.variables.scope">variable scope</link>.
```
Comment (Girgias) [inline language/constants.xml:64]: This isn't totally correct due to visibility on class constants, but I suppose that's kinda irrelevant here?

File: language/basic-syntax.xml [hunk 2]
```diff
@@ -95,52 +123,17 @@ echo "Last statement";
...
-    PHP is built with the <option>--disable-short-tags</option> configuration.
-   </para>
    <para>
     <note>
      <para>
-      As short tags can be disabled it is recommened to only use the normal
-      tags (<code>&lt;?php ?&gt;</code> and <code>&lt;?= ?&gt;</code>) to
-      maximise compatibility.
+      If PHP is embeded within XML or XHTML the normal PHP
+      <code>&lt;?php ?&gt;</code> must be used to remain complient
```
Comment (cmb69) [inline language/basic-syntax.xml:130]:
```suggestion
      <code>&lt;?php ?&gt;</code> must be used to remain compliant
```

File: language/constants.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
    suggests, that value cannot change during the execution of the
    script (except for <link linkend="language.constants.predefined">
    magic constants</link>, which aren't actually constants).
-   A constant is case-sensitive by default. By convention, constant 
+   Constant are case-sensitive. By convention, constant
```
Comment (cmb69) [inline language/constants.xml:11]: Well, in PHP 7 it is still possible to have case-insensitive constants (`define(…, …, true)` and internal constants can be as well). Also should be "constants are" (plural).

File: language/constants.xml (hunk 1 above)
Comment (cmb69) [reply language/constants.xml:64]: That paragraph might not have been updated regarding constant visibility specifiers. Might make sense to add a note ("(if visible)" or such).

File: language/types/pseudo-types.xml [hunk 4]
```diff
@@ -30,7 +31,7 @@
   </para>
 
  </sect2>
-
+<!-- TODO: Proper page? -->
```
Comment (cmb69) [inline language/types/pseudo-types.xml:34]: Yes; but that can be another commit/PR.

File: language/constants.xml (hunk 3 above)
Comment (Girgias) [reply language/constants.xml:11]: I would have assumed the note below to indicate that prior to PHP 8 constants defined using ``define()`` can be case insensitive would be sufficient, but can reword if needed.

Comment (cmb69) [reply language/constants.xml:11]: Ah, I missed that note. All fine then. :)

Comment (Girgias) [reply language/constants.xml]: I moved the note up as it was on the next page in theory. :)

File: language/constants.xml (hunk 1 above)
Comment (Girgias) [reply language/constants.xml:72]: Note added below, is that ok?

File: language/constants.xml [hunk 5]
```diff
@@ -303,8 +327,7 @@ echo ANIMALS[1]; // outputs "cat"
        <row xml:id="constant.coloncolonclass">
         <entry><constant>ClassName::class</constant></entry>
```
Comment (cmb69) [inline language/constants.xml:328]: Maybe
```suggestion
        <entry><constant><replaceable>ClassName</replaceable>::class</constant></entry>
```
Not sure, though, whether that would "work" with PhD.

**Review by cmb69 — APPROVED** [review]
Thanks! Looks good to me.
